src/App.tsx

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type Props = {};
+type State = {
+  pressedKey: string
+};
+
+export class App extends React.Component<Props, State> {
```

Comment:
```suggestion
export class App extends React.Component<{}, State> {
```
You can just put empty object here and remove the `type Props = {};` line